bid.proto

```diff
@@ -15,25 +15,23 @@ message Bid {
     JubJubCompressed commitment = 5;
     BlsScalar elegibility_ts = 6;
     BlsScalar expiration_ts = 7;
-    uint64 bid_tree_storage_index = 8;
+    fixed64 bid_tree_storage_index = 8;
```

Comment:
Anyway, this was not added by you so we should file an issue for it.

I'm curious about why you switched from uint to fixed.

Reply:
You're right, I will revert the change and add an issue for it.